src/main/java/appeng/items/tools/powered/WirelessTerminalItem.java

```diff
@@ -117,7 +137,10 @@ public boolean hasPower(final Player player, final double amt, final ItemStack i
         }
 
         @Override
-        public IConfigManager getConfigManager(final ItemStack target) {
+        public IConfigManager getConfigManager(final ItemStack target) {// TODO maybe provide an easy way for other
+                                                                        // Terminals to overwrite this without making
+                                                                        // their own IWirelessTerminalHandler that
+                                                                        // mostly copies this one
```

Comment:
Can you remove the TODO and make an issue for it? Otherwise this TODO may still be here by 2025 :D